stdlib/3/collections/abc.pyi

```diff
@@ -10,6 +10,7 @@ if sys.version_info >= (3, 3):
         Iterable as Iterable,
         Iterator as Iterator,
         Sized as Sized,
+        Callable as Callable,
```

Comment:
I'm not sure this is right, since typing.Callable is rather different at runtime from collections.abc.Callable.